docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -68,24 +77,61 @@ Ensure that the following prerequisites are met:
 
 - Client Application configuration in the OIDC provider
```

Comment:
Not necessary to be a a bullet point as it is the only one on this level. Maybe update the heading?